### ISSUE TITLE
Can't import curry

### ISSUE THREAD

**Issue by ndavydovdev** [dry-python/returns#378]
I downloaded returns lib by `pip install returns` and there is no curry module. I read docs, searched in previous issues and in StackOverflow, and didn't find how to enable or download it :( Am I doing something wrong?

**Comment by thepabloaguilar** [comment]
Hi @ndavydovdev, `@curry` will be available on version `0.14.0` (WIP yet). The latest version on PyPI is `0.13.0`.
About the documentation, you're seeing the `latest` version that is the latest master branch commit, to see the documentation of current stable version on PyPI please use this link below:
https://returns.readthedocs.io/en/stable/

But if you want to use `@curry` you can install from source code using the following command:
```sh
pip install git+git://github.com/dry-python/returns.git#egg=returns
```
**Caution with that, maybe the master source is not stable at all and it can change!**

**Comment by ndavydovdev** [comment]
Thank you, @thepabloaguilar!
